docs/content/en/docs/getting-started/_index.md

```diff
@@ -15,58 +25,266 @@ In your local environment the following tools are required for provisioning a K8
+
+* 1.16
+* 1.17
+* 1.18
+* 1.19
+* 1.20
+
+### Cassandra version support
+
+K8ssandra can install the following versions of Apache Cassandra:
 
-K8ssandra is delivered as a collection of Helm Charts. In order to leverage these charts we have provided a k8ssandra Helm Repository for easy installation. 
+* 3.11.7
+* 3.11.8
+* 3.11.9
+* 3.11.10
+* 4.0-beta4
 
-Also add the Traefik Ingress repo - you'll need its resources to access services from outside the Kubernetes cluster.
+### Resource recommendations
 
-```console
-helm repo add k8ssandra https://helm.k8ssandra.io/
-helm repo add traefik https://helm.traefik.io/traefik
-helm repo update
+The minimum recommended development configuration for a single Cassandra node is 8 gigs of RAM and 2 virtual processor cores (2 physical cores). Given that, we recommend a machine specification of **no less** than 16 gigs of RAM and 8 virtual processor cores (4 physical cores). You'll want to adjust your Docker resource preferences accordingly. For this quick start we're allocating 4 virtual processors and 8 gigs of RAM to the Docker environment.
+
+An ideal environment, enabling you to run a 3 node Cassandra cluster, enforcing QUORUM consistency, would consist of 32 gigs of RAM and 12 virtual cores (6 physical cores).
+
+See the documentation for your particular flavor of Docker for instructions on configuring resource limits.
+
+### Example K8s container configuration
+
+The following Minikube example creates a K8s cluster named `k8ssandra` running K8s version 1.18.16 with 4 virtual processor cores and 8 gigs of RAM:
+
+```bash
+minikube start --cpus=4 --memory='8128m' --kubernetes-version=1.18.16 -p k8ssandra
+😄  [k8ssandra] minikube v1.17.1 on Darwin 11.2.1
+✨  Automatically selected the docker driver. Other choices: hyperkit, ssh
+👍  Starting control plane node k8ssandra in cluster k8ssandra
+🔥  Creating docker container (CPUs=4, Memory=8128MB) ...
+🐳  Preparing Kubernetes v1.18.16 on Docker 20.10.2 ...
+    ▪ Generating certificates and keys ...
+    ▪ Booting up control plane ...
+    ▪ Configuring RBAC rules ...
+🔎  Verifying Kubernetes components...
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "k8ssandra" cluster and "default" namespace by default
 ```
 
-Alternatively, you may download the individual charts directly from the project's [releases](https://github.com/k8ssandra/k8ssandra/releases) page.
+## Configure the K8ssandra Helm repository
+
+K8ssandra is delivered via a collection of Helm charts for easy installation, so once you've got a suitable K8s environment configured, you'll need to add the K8ssandra Helm chart repositories.
+
+To add the K8ssandra helm chart repos:
+
+1. Add the main K8ssandra Helm chart repo:
+
+    ```bash
+    helm repo add k8ssandra https://helm.k8ssandra.io/
+    ```
+
+2. If you want to access K8ssandra services from outside of the Kubernetes cluster, also add the Traefik Ingress repo (highly recommended):
```

Comment:
Do we want to include Traefik here? Depending on your use case it may not be required.

Reply:
(Read: Should this be moved to the DevOps section and leave this part of the path?)

Reply:
I'm sort of ambivalent on this. Honestly I don't see why we don't just glom the K8ssandra and Traefik repos into one. It's not like having the traefik repos will harm anything or get in the way.